ios/StripeTerminalReactNative.swift

```diff
@@ -170,10 +170,11 @@ class StripeTerminalReactNative: RCTEventEmitter, DiscoveryDelegate, BluetoothRe
     func discoverReaders(params: NSDictionary, resolver resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) {
         let simulated = params["simulated"] as? Bool
         let discoveryMethod = params["discoveryMethod"] as? String
-
+        let timeout = params["timeout"] as? UInt ?? 0
```

Comment:
@bric-stripe any issues with setting the timeout to 0 by default?

Reply:
nope, no concerns. that's the native SDKs default as well 👍